packages/broker/src/plugins/metrics/node/MetricsPublisher.ts

```diff
@@ -72,21 +72,24 @@ export class MetricsPublisher {
         const stream = await this.client.getOrCreateStream({
             id: this.getStreamId(periodLegth)
         })
-        // TODO: pretify this error handler
-        // https://linear.app/streamr/issue/BACK-155/assign-a-stream-to-a-storage-node-when-it-has-already-been-assigned
-        try {
-            await stream.addToStorageNode(this.storageNodeAddress)
-        } catch (e) {
-            if (!e.body) { throw e }
-            let parsedBody
+
+        if (periodLegth > PERIOD_LENGTHS.FIVE_SECONDS) {
```

Comment:
Could use exact comparison `(periodLength !== PERIOD_LENGTHS.FIVE_SECONDS)` as that is the only exception and in this scope we are comparing two IDs.